bentoctl/cli/utils.py

```diff
@@ -1,13 +1,20 @@
 import functools
 import os
+import logging
 
 import cloup
 
 DEBUG_ENV_VAR = "BENTOCTL_DEBUG"
 
 
-def set_debug_mode(enabled: bool):
-    os.environ[DEBUG_ENV_VAR] = str(enabled)
+def set_debug_mode(is_enabled: bool):
+    if is_enabled or os.environ.get(DEBUG_ENV_VAR):
+        os.environ[DEBUG_ENV_VAR] = str(True)
+        logging.getLogger().setLevel(logging.DEBUG)
+        logging.getLogger("bentoml").setLevel(logging.DEBUG)
```

Comment:
let's not use the bentoml logger. It might interfere user's bentoml setting.

Also, where is the logger from, did we set it somewhere?

Reply:
I mean let's not use the bentoml namespace.

Reply:
resolved offline. Solution is add comment for this